src/main/resources/database/changelog-master.yml

```diff
@@ -16,7 +16,9 @@ databaseChangeLog:
       file: database/changes/release-10.49.0/changelog.yml
   - include:
       file: database/changes/release-10.49.1/changelog.yml
-      
+  - include:
+      file: database/changes/release-10.49.16/changelog.yml
```

Comment:
Can this be changed to follow the format as the comment shows (release-11)? The reason for this change is to avoid confusion of the migration scripts matching the app release version (which was not followed in a number of repositories).